tests/test_backend.py

```diff
@@ -19,13 +19,26 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import logging
+
 import onnx.backend.test
 import pytest
 
 import ngraph_onnx.onnx_importer.backend as ng_backend
 
 import tests.utils
 
+
+def expect_fail(test_path):  # type: (str) -> None
```

Comment:
I'm not sure that `test_path` is a correct name for it.
Maybe `test_case` is better?

Reply:
Well, I'm trying to refer to Python's full `import` module dotted path, such as `Module.test_case`.

`test_case` would indicate that we need a `unittest.TestCase` or another object.

As far as I know these paths don't have a proper name in Python. I even asked on SO:
https://stackoverflow.com/q/56358544/194905

Maybe `test_case_path` or `test_case_name` would be better?

Reply:
`test_case_path` sounds good to me.